packages/aws-cdk/lib/commands/docs.ts

```diff
@@ -26,8 +26,7 @@ export function handler(args: yargs.Arguments) {
 }
 
 export async function realHandler(options: CommandOptions): Promise<number> {
-  const docVersion = require('../../package.json').version;
-  const url = `https://awslabs.github.io/aws-cdk/versions/${docVersion}/`;
+  const url = `https://docs.aws.amazon.com/cdk/api/latest/`;
```

Comment:
Why not the correct version? (or do we still don't have that, in which case open an issue so we don't forget).